kofta/src/index.css

```diff
@@ -63,3 +63,8 @@ option {
   background: none;
   border: none;
 }
+
+::selection {
+  background-color: black;
```

Comment:
i believe @benawad doesn't want custom selector rn. but he said that a while ago so i may be wrong